pkg/util/log/clog.go

```diff
@@ -814,12 +814,17 @@ func (l *loggingT) outputLogEntry(s Severity, file string, line int, msg string)
 		// https://github.com/cockroachdb/cockroach/issues/23119
 		fatalTrigger = make(chan struct{})
 		exitFunc := l.exitFunc
+		exitCalled := make(chan struct{})
```

Comment:
This fixes an issue where calling `exitFunc` is delayed after calling `log.Fatal`